hi-ml-azure/src/health_azure/utils.py

```diff
@@ -65,6 +65,15 @@
 ENV_LOCAL_RANK = "LOCAL_RANK"
 MASTER_PORT_DEFAULT = 6105
 
+# Environment variables used by Amulet
+ENV_AMLT_PROJECT_NAME = "AZUREML_ARM_PROJECT_NAME"
+ENV_AMLT_INPUT_OUTPUT = "AZURE_ML_INPUT_OUTPUT"
```

Comment:
Minor - for consistency should we use `
``suggestion
ENV_AMLT_INPUT_OUTPUT = "AZUREML_INPUT_OUTPUT"
```
?

Reply:
but those variables are defined by amulet?

Reply:
This env var is set by Amulet  - we just read it